src/pybind/mgr/prometheus/module.py

```diff
@@ -385,13 +395,24 @@ def get_osd_stats(self):
                 self.metrics.append('osd_{}'.format(stat), val,
                                     ('osd.{}'.format(id_),))
 
+    def get_service_list(self):
+        ret = {}
+        for server in self.list_servers():
+            version = server.get('ceph_version', '')
```

Comment:
Can we parse the version a bit? The entire string looks like this

```ceph version 12.2.2-g123456 (<full_commid_id>) luminous (stable)```

and it is just way too long. Removing `ceph version ` and splitting by first space like I did in my PR should be fairly safe and it would show a reasonable representation of the version.

Reply:
I would rather not remove this information at the exporter level. The ```ceph version``` string seems superfluous to me and if so it should be removed at the mgr level. The rest of the version string carries information. If its not wanted that can always be removed at a higher level (say Grafana).